2016/figures.html

```diff
@@ -44,7 +44,7 @@
 
     <div class="stat">
       <p class="stat__main">
-        <span class="stat__number">[?????]</span>
+        <span class="stat__number">45,371</span>
```

Comment:
I think this should be adde to the EP number unless we’re going to remove the EP text just below it.